### ISSUE TITLE
SecurityError: The operation is insecure.

### ISSUE THREAD

**Issue by S1SYPHOS** [ganlanyuan/tiny-slider#70]
__Issue description__:  
`if(styleSheet.cssRules)`, found in `getCssRulesLength.js` helper, invokes "SecurityError: The operation is insecure.", same for current Chrome. Seems to be something with loading styles from others domains, kind of a CSP-related issue (maybe [this](https://stackoverflow.com/questions/21642277/security-error-the-operation-is-insecure-in-firefox-document-stylesheets) is a good starting point). Maybe go for a  try / catch?

__Demo link/slider setting__:   
```js
import { tns } from 'tiny-slider/src/tiny-slider.module';

function tinySlider() {
  let slider = tns({
    container: '.slider',
    controlsContainer: '.slider-controls',
    gutter: 10,

    slideBy: 1,
    speed: 1250,

    autoplay: true,
    autoplayTimeout: 3500,
    autoplayHoverPause: true,

    responsive: {
      '0': 1,
      '600': 2,
      '900': 3
    }
  });
}

tinySlider();
```

_Tiny-slider version_:   2.1.6
_Browser name && version_:  FF 55
_OS name && version_:   Fedora 26

**Comment by ganlanyuan** [comment]
Hi, I couldn't reproduce this. (I'm on mac os)
I just update the `getCssRulesLength` function in [this commit](https://github.com/ganlanyuan/tiny-slider/tree/ff398107e5176998cb9916854b4ea6649427a6d8), could you help test it?

**Comment by S1SYPHOS** [comment]
`SecuritySuccess: The operation is secure.`

**Comment by ganlanyuan** [comment]
Okay, thanks

**Comment by ganlanyuan** [comment]
The fix was added in [v2.1.7](https://github.com/ganlanyuan/tiny-slider/releases/tag/v2.1.7)
